packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts

```diff
@@ -515,17 +515,17 @@ export const StaticProperties: ObjectMap2<CoreJSPolyfillDescriptor> = {
 
   Promise: {
     all: define(null, PromiseDependenciesWithIterators),
-    allSettled: define(null, [
+    allSettled: define("promise/all-settled", [
       "es.promise.all-settled",
       ...PromiseDependenciesWithIterators,
     ]),
-    any: define(null, [
+    any: define("promise/any", [
       "es.promise.any",
       "es.aggregate-error",
       ...PromiseDependenciesWithIterators,
     ]),
     race: define(null, PromiseDependenciesWithIterators),
-    try: define(null, ["esnext.promise.try", ...PromiseDependencies]),
+    try: define("promise/try", ["esnext.promise.try", ...PromiseDependencies]),
     withResolvers: define(null, [
```

Comment:
It's also a static method - it was added recently, but with `Map.groupBy` situation the same.